vlm/src/main/scala/geotrellis/contrib/vlm/GeoTiffReprojectRasterSource.scala

```diff
@@ -28,15 +28,15 @@ class GeoTiffReprojectRasterSource(
   val uri: String,
   val crs: CRS,
   val options: Reproject.Options = Reproject.Options.DEFAULT
-) extends RasterSource {
-  @transient private lazy val tiff: MultibandGeoTiff =
+) extends RasterSource { self =>
+  @transient lazy val tiff: MultibandGeoTiff =
     GeoTiffReader.readMultiband(getByteReader(uri), streaming = true)
 
-  private lazy val baseCRS = tiff.crs
-  private lazy val baseRasterExtent = tiff.rasterExtent
```

Comment:
Why remove all the `private` ? Not a problem now but eventually I could see this becoming part of the API unintentionally.

Reply:
Ah will fix it, forgot to revert changes after the debug process.